com.ultimaker.cura.yml

```
@@ -0,0 +1,313 @@
+
+  - name: arcus
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DCMAKE_INSTALL_PREFIX=/app
+      - -DCMAKE_PREFIX_PATH=/app
+      - -DCMAKE_CXX_STANDARD=17
+      - -DBUILD_STATIC=ON
+      - -DBUILD_PYTHON=ON
+      - -DBUILD_EXAMPLES=OFF
+    sources:
+      - type: git
+        url: https://github.com/ultimaker/libArcus.git
+        commit: 2066a9ddccece8f970902bb5e33147d674627edd
+        tag: 4.13.0 
+      - type: patch
+        path: patch/libarcus-python-binindg-install.patch
+
+  - name: pynest2d
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DCMAKE_PREFIX_PATH=/app
+      - -DCMAKE_INSTALL_PREFIX=/app
+      - -DCMAKE_CXX_STANDARD=17
+      - -DBUILD_STATIC=ON
+    sources:
+      - type: git
+        url: https://github.com/Ultimaker/pynest2d.git
+        commit: baec470ec51f334f6bc5b275ddb946083970aac5
+        tag: 4.12.1
+      - type: patch
+        path: patch/pynest2d-python-install.patch
+
+  - name: savitar
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DCMAKE_PREFIX_PATH=/app
+      - -DCMAKE_INSTALL_PREFIX=/app
+      - -DCMAKE_CXX_STANDARD=17
+      - -DBUILD_STATIC=ON
+    sources:
+      - type: git
+        url: https://github.com/ultimaker/libSavitar.git
+        commit: 68a8868e0b30883b461c0dde77f6e85ce0a4d2ef
+        tag: 4.13.0
+      - type: patch
+        path: patch/libsavitar-python-install.patch
+
+  - name: libCharon
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_PREFIX_PATH=/app
+      - -DCMAKE_INSTALL_PREFIX=/app
+      - -DPYTHON_EXECUTABLE=/app/bin/python3
+    sources:
+      - type: git
+        url: https://github.com/Ultimaker/libCharon
+        commit: 8e255aaad10b09c9cfc6e795f33968fd642e26c9
+        tag: 4.13.0
+
+  - name: cura-engine
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DCMAKE_PREFIX_PATH=/app
+      - -DCMAKE_INSTALL_PREFIX=/app
+      - -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON
+      - -DCMAKE_CXX_STANDARD=17
+      - -DCURA_ENGINE_VERSION=4.13.0
+      - -DENABLE_MORE_COMPILER_OPTIMIZATION_FLAGS=ON
+    sources:
+      - type: git
+        url: https://github.com/ultimaker/CuraEngine
+        commit: 0e79559492a9e3e631c28624b73a888b3297f174
+        tag: 4.13.0
+
+
+  - name: uranium
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_PREFIX_PATH=/app
+      - -DCMAKE_INSTALL_PREFIX=/app
+    sources:
+      - type: git
+        url: https://github.com/ultimaker/Uranium
+        commit: b1d9219a7272140a2021e7339901897c7df927d5
+        tag: 4.13.0
+
+  - name: fdm_materials
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_PREFIX_PATH=/app
+      - -DCMAKE_INSTALL_PREFIX=/app
+    sources:
+      - type: git
+        url: https://github.com/ultimaker/fdm_materials
+        commit: 304287c3a69d79a14d33c1fe897bf653a2fc4f15
+        tag: 4.13.0
+
+  - name: cura
+    buildsystem: cmake
```

```suggestion
    buildsystem: cmake-ninja
```

unless it fails, it's faster